package/ddvk-hacks/package

```diff
@@ -0,0 +1,200 @@
+            patch_version="17.1.04"
+            device="reMarkable 1"
+            original_hash="4296b9c6d7a66aadd12e1cf61a13b7b19504673d"
+            xochitl_version="2.5.0.27"
+            ;;
+        "20201028164335")
+            patch_version="16.1.06"
+            device="reMarkable 1"
+            original_hash="336529ce6e7ef9d6fadd30872708556ca8711f0b"
+            xochitl_version="2.4.1.30"
+            ;;
+        "20201028163830")
+            patch_version="16.2.03"
+            device="reMarkable 2"
+            original_hash="c88d155b7ca8c770240b2c00048968f8445f8115"
+            xochitl_version="2.4.1.30"
+            ;;
+        "20201016123042")
+            patch_version="15.2.01"
+            device="reMarkable 2"
+            original_hash="797f58ed93d2e22e7d77fcd9de6c6eb5d49a3a7f"
+            xochitl_version="2.4.0.27"
+            ;;
+        "20201016123325")
+            patch_version="15.1.02"
+            device="reMarkable 1"
+            original_hash="891e06535c0ae742eeaa3b9a20e9ff03d0f659d3"
+            xochitl_version="2.4.0.27"
+            ;;
+        "20200914085553" | "20200914090635")
+            patch_version="14.01"
+            device="reMarkable 2"
+            original_hash="596b02f401fb0ceb6a73df470fbab418b305cdbc"
+            xochitl_version="2.3.1.27"
+            ;;
+        "20200904144143")
+            patch_version="13.07"
+            device="reMarkable 2"
+            original_hash="7eb1ed8b75b1b282fd4ecf30ef19118d3a41fcc7"
+            xochitl_version="2.3.0.23"
+            ;;
+        "20200709160645")
+            patch_version="12.11"
+            device="reMarkable 1"
+            original_hash="005b05ef64f079aaf377d373cb7e2889a2aa774a"
+            xochitl_version="2.3.0.16"
+            ;;
+        "20200805214933")
+            patch_version="11.01"
+            device="reMarkable 2"
+            original_hash="c7d965972a5a6d2bf8503b1b09b52a89c422505b"
+            xochitl_version="2.2.1.82"
+            ;;
+        "20200528081414")
+            patch_version="10.10"
+            device="reMarkable 1"
+            original_hash="7e92c177df685972a699db6c4a7a918296447f74"
+            xochitl_version="2.2.0.48"
+            ;;
+        "20200320131825")
+            patch_version="09"
+            device="reMarkable 1"
+            original_hash="c8661fbd74a049134509dc22da415bb651d7feac"
+            xochitl_version="2.1.1.3"
+            ;;
+        *)
+            echo
+            echo "The version the device is running is not supported, yet."
+            echo "    Build date: $build_date"
+            echo
+            exit 1
+            ;;
+    esac
+
+    echo
+    echo "Device: $device"
+    echo "Xochitl version: $xochitl_version"
+    echo "Patch version: $patch_version"
+    echo
+
+    if ! sha1sum -c <(echo "$original_hash  $_xochitl_path") > /dev/null 2>&1; then
+        echo "Invalid Xochitl checksum"
+        echo "Maybe ddvk-hacks are already installed?"
```

Comment:
Perhaps check the backup location that xochitl gets placed in when using @ddvk's script?

Reply:
Good point. If we detect its presence, should we just warn the user and ask them to do the cleanup? Or use the upstream script’s backup to restore Xochitl, then re-run our script as normal?

Reply:
Not quite sure. @ddvk what are you comfortable with?

Reply:
probably it is better to let the user do the cleanup